mypy.ini

```diff
@@ -1,3 +1,4 @@
 [mypy]
-mypy_path = ./third-party-stubs
+mypy_path = ./third-party-stubs,./core
```

Comment:
If you still have it handy - what was the message from the error that this resolved? I'm curious why we don't need to do the same for later versions of core

Reply:
I think we added or removed an init file in the later versions. I can't recall verbatim, but the issue was that files could be found via two paths, one starting at `/dbt` and another starting at `/core`; it seems like these were both in $PATH. So it was something like this:

```
Module found with multiple paths: `/core/dbt/adapters/base/meta.py` and `/dbt/adapters/base/meta.py`.
```

But a side effect seemed to be that we were not type checking `/core/dbt/` at all. Because once I turned it on, I got 46 errors in 30 files (I guess I remember numbers better than words).